core/controllers/collection_editor_test.py

```diff
@@ -99,6 +100,68 @@ def test_access_collection_editor_page(self):
         self.assert_can_edit(response.body)
         self.logout()
 
+
+    def test_editable_collection_handler_get(self):
+        whitelisted_usernames = [self.EDITOR_USERNAME]
+        self.set_config_property(
+            config_domain.WHITELISTED_COLLECTION_EDITOR_USERNAMES,
+            whitelisted_usernames)
+
+        # Check that non-editors cannot access the editor data handler.
+        # This is due to them not being whitelisted.
+        response = self.testapp.get(
+            '%s/%s' % (feconf.EDITABLE_COLLECTION_DATA_URL_PREFIX,
+                       self.COLLECTION_ID))
+        self.assertEqual(response.status_int, 302)
+
+        # Check that whitelisted users can access the data
+        # from the editable_collection_data_handler
+        self.login(self.EDITOR_EMAIL)
+
+        json_response = self.get_json(
+            '%s/%s' % (feconf.EDITABLE_COLLECTION_DATA_URL_PREFIX,
+                       self.COLLECTION_ID))
+        self.assertEqual(self.COLLECTION_ID, json_response['collection_id'])
+        self.assertEqual(
+            'Introduction to Collections in Oppia', json_response['title'])
+
+        self.logout()
+
+
+    def test_editable_collection_handler_put(self):
+        whitelisted_usernames = [self.EDITOR_USERNAME]
+        self.set_config_property(
+            config_domain.WHITELISTED_COLLECTION_EDITOR_USERNAMES,
+            whitelisted_usernames)
+
+        # Check that whitelisted users can access the data
+        # from the editable_collection_data_handler
+        self.login(self.EDITOR_EMAIL)
+
+        # Call create handler to return csrf token
+        response = self.testapp.get(
+            '%s/%s' % (feconf.COLLECTION_EDITOR_URL_PREFIX,
+                       self.COLLECTION_ID))
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        json_dict = {"version" : 1,
```

Comment:
Consider formatting a bit more consistently with the rest of the codebase:

```python
json_dict = {
    'version' : 1,
    'commit_message' : 'changed title',
    'change_list' : [{
        'cmd': 'edit_collection_property',
        'property_name': 'title',
        'new_value': 'A new title'
    }]
}
```

Also, we only use single quotes for strings.